android/app/src/main/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This file shouldn't be in Git 😵

Reply:
Yeah, it currently is though, will need to look into how to keep these more secure, but if someone REALLY wanted to I believe they could get this info from the compiled app